assets/css/_drawer_tab.scss

```diff
@@ -18,5 +18,5 @@
   font-size: 2rem;
   line-height: 1.5rem;
   margin: 0;
-  padding: 2rem 0;
+  padding: 2rem 0 !important;
```

Comment:
note(non-blocking): while on my-charlie, we ended up using a lot of bootstrap/tailwind-like utilities, and bootstrap already has `px-2` and `py-0` ([padding x-axis value 2, and padding y axis value 0](https://getbootstrap.com/docs/5.3/utilities/spacing/))

So ideally we could standardize with design on what kind of value they want instead of hard coding it per component.

I'm on the fence as to whether this would be good for skate, but I do like the value bootstrap utilities bring to "DRYing" out CSS and removing repetition (like `!important`). I also don't know that we've _decided_ on using bootstrap as a design library, so I only bring this up as a learning and not anything we need to act on right now.

(if you wanted to try this out though, you'd need to import these files to `app.scss`)
```
@import "../node_modules/bootstrap/scss/utilities";
@import "../node_modules/bootstrap/scss/utilities/api";
```